.controlplane/entrypoint.sh

```
@@ -0,0 +1,26 @@
+  echo " -- $1 is available"
+}
+
+echo " -- Starting entrypoint.sh"
+
+# Wait for Postgres if DATABASE_URL is set
+if [ -n "$DATABASE_URL" ]; then
+  echo " -- Waiting for Postgres"
+  wait_for_service $(echo $DATABASE_URL | sed -e 's|^.*@||' -e 's|/.*$||')
+fi
+
+echo " -- Finishing entrypoint.sh, executing '$@'"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)